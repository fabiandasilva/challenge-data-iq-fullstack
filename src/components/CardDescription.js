import React from "react";
const CardDescription = () => {
  return (
    <>
      <div className="px-4 sm:px-0 ">
        <h3 className="text-base font-semibold leading-7 text-gray-900">
          Informaci√≥n personal
        </h3>
        <p className="mt-1 max-w-2xl text-sm leading-6 text-gray-500">
          Datos personales del usuario
        </p>
      </div>
      <div className="mt-6 border-t border-gray-100">
        <dl className="divide-y divide-gray-100">
          <div className="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0">
            <dt className="text-sm font-medium leading-6 text-gray-900">
              Nombre
            </dt>
            <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
              Margot Foster
            </dd>
          </div>
          <div className="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0">
            <dt className="text-sm font-medium leading-6 text-gray-900">
              Usuario
            </dt>
            <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
              Backend Developer
            </dd>
          </div>
          <div className="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0">
            <dt className="text-sm font-medium leading-6 text-gray-900">
              Email
            </dt>
            <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
              margotfoster@example.com
            </dd>
          </div>
          <div className="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0">
            <dt className="text-sm font-medium leading-6 text-gray-900">
              Ciudad
            </dt>
            <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
              $120,000
            </dd>
          </div>
          <div className="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0">
            <dt className="text-sm font-medium leading-6 text-gray-900">
              Empresa
            </dt>
            <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
              $120,000
            </dd>
          </div>
        </dl>
      </div>
    </>
  );
};

export default CardDescription;
