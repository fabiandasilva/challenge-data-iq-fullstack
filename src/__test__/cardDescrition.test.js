/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import React from "react";
import CardDescription from "../components/CardDescription";
import userDetail from "./mock/mockDetail";
test("CardDescription se renderiza correctamente", () => {
  const { asFragment } = render(<CardDescription userDetail={userDetail} />);

  expect(asFragment()).toMatchSnapshot();
});
