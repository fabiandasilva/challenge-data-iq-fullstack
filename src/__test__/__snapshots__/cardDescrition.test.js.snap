// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDescription se renderiza correctamente 1`] = `
<DocumentFragment>
  <div
    class="px-4 sm:px-0 "
  >
    <h3
      class="text-base font-semibold leading-7 text-gray-900"
    >
      Informaci√≥n personal
    </h3>
    <p
      class="mt-1 max-w-2xl text-sm leading-6 text-gray-500"
    >
      Datos personales del usuario
    </p>
  </div>
  <div
    class="mt-6 border-t border-gray-100"
  >
    <dl
      class="divide-y divide-gray-100"
    >
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Nombre
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          Nombre de ejemplo
        </dd>
      </div>
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Usuario
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          @ejemplo
        </dd>
      </div>
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Email
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          ejemplo@example.com
        </dd>
      </div>
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Ciudad
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          Ciudad de ejemplo
        </dd>
      </div>
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Empresa
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          Empresa de ejemplo
        </dd>
      </div>
      <div
        class="px-4 py-6 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-0"
      >
        <dt
          class="text-sm font-medium leading-6 text-gray-900"
        >
          Pagina web
        </dt>
        <dd
          class="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0"
        >
          https://www.ejemplo.com
        </dd>
      </div>
    </dl>
  </div>
</DocumentFragment>
`;
